common/flatpak-installation.c

```diff
@@ -3071,10 +3071,11 @@ flatpak_installation_list_unused_refs (FlatpakInstallation *self,
  *
  *   * exclude-refs (as): Act as if these refs are not installed even if they
  *       are when determining the set of unused refs
- *   * filter-by-eol (b): Only return refs as unused if they are End-Of-Life.
- *       Note that if this option is combined with other filters (of which there
- *       are none currently) non-EOL refs may also be returned.
- *
+ *   * filter-by-eol (b): Return refs as unused if they are End-Of-Life.
+ *       Note that if this option is combined with other filters so non-EOL refs may also be returned.
+ *   * filter-by-autoprune (b): Return refs as unused if they should be autopruned.
+ *       Note that if this option is combined with other filters so non-autoprune refs may also be returned.
```

Comment:
The wording seems odd here: it seems like you are not sure whether to say "note that (fact) so (consequence)" or "note that if (possible situation), (consequence)".

I think perhaps you mean the latter, in which case I think it would be clearer written as "note that if (possible situation)", then (consequence)":

```suggestion
 *   * filter-by-eol (b): Return refs as unused if they are End-Of-Life.
 *       Note that if this option is combined with other filters, then non-EOL refs may also be returned.
 *   * filter-by-autoprune (b): Return refs as unused if they should be autopruned.
 *       Note that if this option is combined with other filters, then non-autoprune refs may also be returned.
```

Or if you meant "note that (fact) so (consequence)" then maybe it would be clearer as "note that (fact), therefore (consequence)".